pyDeltaRCM/deltaRCM_tools.py

```diff
@@ -8,15 +8,16 @@
 
 from scipy.sparse import lil_matrix, csc_matrix, hstack
 
-from .sed_tools import sed_tools
-from .water_tools import water_tools
-from .init_tools import init_tools
-from .debug_tools import debug_tools
-from .shared_tools import get_random_state
+from . import shared_tools
 
 
-class Tools(sed_tools, water_tools, init_tools, debug_tools, object):
-    """Aggregate all `*_tools` modules into single class, define api methods."""
+class iteration_tools(object):
+    """Tools relating to the updating of the model and model I/O.
+
+    Tools defined in this class include steps to iterate for one timestep,
+    finalize timesteps, and saving output figures, grids, and checkpoints.
+    Additionally, most stratigraphy-related operations are defined here.
+    """
```

Comment:
I think, it's fine because we never instantiate the class directly. I guess we could maybe call it an `@abstractclass`, which would prevent anyone from instantiating it anyway, potentially giving a better error message? I'll look into it.

Reply:
I changed these to all be `abc.ABC` types, so they cannot be instantiated directly.